scripts/split.sh

```diff
@@ -46,19 +46,27 @@ do
   shift
 done
 
+if [ -z "$LC0LOCKFILE" ]
+then
+  echo "env var LC0LOCKFILE not set"
+  exit 1
+fi
 
 # clear test and train split dirs
-rm -rf $TESTDIR $TRAINDIR
-mkdir -vp $TESTDIR $TRAINDIR
+if [ ! -d "$TESTDIR" ] || [ ! -d "$TRAINDIR" ]
+then
+  rm -rf $TESTDIR $TRAINDIR
```

Comment:
Maybe quote around $TESTDIR and $TRAINDIR here - I'm always wary of doing an rm -rf without proper quoting.